src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```diff
@@ -435,23 +412,23 @@ const ImageClassificationObservationTable = ({
                         <TdWithHoverText
                           data-tooltip={file.original_image_name}
                           onClick={() => handleImageClick(file)}
-                          cursor={file.classification_status.status === 3 ? 'pointer' : 'default'}
+                          cursor={file.classification_status?.status === 3 ? 'pointer' : 'default'}
```

Comment:
I know this isnt in the scope of this diff which is just adding an `?.`, but a dev cant tell what `3` signifies here. I like to use enums or objects to make things like this more self documenting. 

eg:

```
const CLASSIFICATION_STATUSES { 
	BUMPY: 3,
	SMOOTH: 4,
	STICKY: 5,
}

...

cursor={file.classification_status?.status === CLASSIFICATION_STATUSES.sticky ? 'pointer' : 'default'}
```